py.typed not included in PyPI distribution

### What action do you want to perform

I'm attempting to import submodules of `pytest_redis` within my project, using version 3.0.1. Something like:
```python
import pytest_redis.executor
import pytest_redis.factories
```

### What are the results

When I run a `mypy` check on this file, I get the following errors:
```
tests/conftest.py:15: error: Skipping analyzing "pytest_redis.executor": module is installed, but missing library stubs or py.typed marker  [import]
    import pytest_redis.executor
    ^
tests/conftest.py:15: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
tests/conftest.py:15: error: Skipping analyzing "pytest_redis": module is installed, but missing library stubs or py.typed marker  [import]
    import pytest_redis.executor
    ^
tests/conftest.py:16: error: Skipping analyzing "pytest_redis.factories": module is installed, but missing library stubs or py.typed marker  [import]
    import pytest_redis.factories
```


### What are the expected results

This seemed strange to me, since I noticed that this project has a [py.typed](https://github.com/ClearcodeHQ/pytest-redis/blob/b3508f9d13a562aa6355ca6b0cb3a8bb18817a05/pytest_redis/py.typed) file which _should_ inform Mypy that this repo has type annotations in place. But when I looked in my local `site-packages/pytest_redis` directory, I see the following:
```
-rw-r--r--  1 max  staff   861 Apr 18 23:46 __init__.py
drwxr-xr-x  7 max  staff   224 Apr 19 01:08 __pycache__
-rw-r--r--  1 max  staff  1527 Apr 18 23:46 config.py
-rw-r--r--  1 max  staff   343 Apr 18 23:46 exception.py
drwxr-xr-x  6 max  staff   192 Apr 18 23:49 executor
drwxr-xr-x  7 max  staff   224 Apr 18 23:49 factories
-rw-r--r--  1 max  staff  5021 Apr 18 23:46 plugin.py
```
The lack of a local `py.typed` file misleads Mypy into believing that this package does not have type annotations, which is why it gives me those errors. When I manually touch a file with that name, Mypy then passes without error.

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,3 +1,3 @@
-include *.rst *.py
+include *.rst *.py pytest_redis/py.typed
 recursive-include src/pytest_redis *.py
 prune docs/build

--- a/newsfragments/471.bugfix.rst
+++ b/newsfragments/471.bugfix.rst
@@ -1,0 +1,1 @@
+Include py.typed in MANIFEST.in

